src/stories/components/MainMenu/styles.scss

```diff
@@ -0,0 +1,33 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Main Menu
+
+// .main-menu {}
+
+.menu-item {
+    min-width: 200px;
+    max-width: 200px;
+	padding: 12px 0 12px 35px;
+	display: flex;
+	align-items: center;
+    cursor: pointer;
+    font-weight:normal!important; 
+    margin-top: 15px;
+    margin-bottom: 15px;
+}
+
+.menu-item:hover {
+    background: linear-gradient(to right, $teal-dark 35%, white 42%);
+    fill: white;
+
+}
+.menu-item:active, .active {
```

Comment:
You should nest the `.active` class inside the `menu-item`. Otherwise, if another element in the site uses the same class it will inherit the styles.

Reply:
I think the .active should be nested inside the menu-item, not the other way around. I saw the updated styles.scss, but I think should be like this:

.menu-item__icon {
    margin-right: 15px;
    padding: 10px 15px 10px 50px;
    display: flex;
    .active {
        background: $teal;
        fill: $white;
        border-top-right-radius: 100px;
        border-bottom-right-radius: 100px;
    }
}

Another thing, maybe we also need to highlight the menu item on hover. I think it'll look better and clearer. As it is not specified in the requirement, maybe you can decide what color goes for the highlighting.

Reply:
I totally agree with that idea - 'active' being nested inside of 'menu-item__icon' and 'menu-item__text'. However I was unable / unsuccessful at executing the styles in that fashion. I believe it does not work in that fashion because 'active' is parent to the icon and text. So the only way to utilize the parent 'active' tag is to have the children nested inside of it. 

I have added hover styling to the component!

Reply:
'active' class is unique feature to react router's NavLink. Instead of having a component with state to keep track of which link is active, NavLink is useful in that it will automatically keep track of which 'link' is active, and adds the 'active' class to that element. This 'active' tag can be renamed via activeClassName but it does not change the structure / architecture of the parent to child relationship.

Reply:
@alecherryy can I get your second opinion on this? I think everything else looks great, just want to make sure about this one. Thanks!

Reply:
Yes please! I am quite new to front end styling so I may be wrong... but that's the conclusion I came to after spending hours of trying to figure it out 😅

Reply:
I think the current implementation is fine, if you look at how the `active` tag is applied, it's applied to the `parent` component of `NavLink` so I don't think you can nest it

Reply:
Update: with the help of Carlo, I have changed the styles so that the active class is nested into the parent element! The active class will not interfere with other components now if an active class is used!

Reply:
That makes sense! Thank you @cmutuc25 for checking!